src/styles/lists/list/index.njk

```diff
@@ -1,5 +1,5 @@
 ---
-title: Lists – Typography
+title: Lists – Lists
```

Comment:
We don't need no repetition.

```suggestion
title: Lists
```

For consistency's sake the name of the example (in the file path) should probably be updated to `default` too.

Reply:
@querkmachine thanks! Renamed the example folders for consistency in 86fadee.